specs/TitleBar/titlebar-functional-spec.md

```diff
@@ -0,0 +1,479 @@
+<!-- The purpose of this document is to describe the design and implementation of a new WinUI control.
+This document contains architectural and implementation details that do not appear in the functional spec. -->
+
+<!-- 
+Author: Karen
+Audiences: Github community contributors to learn about the control, provide feedback 
+
+This PR will be open for feedback for a month, 10/11-11/11
+For more info on the public api review process: https://aka.ms/winappsdk/api-specs-review 
+-->
+
+TitleBar Functional Spec
+===
+*Control category: Basic Input*
+
+# Background
+Motivations for the TitleBar control include:
+
+- Achieve parity with Fluent Windows Visual Design Library.
+- Bridge the gap between Shell / Win32 title bar and a fully custom Xaml titlebar.
+- Simplify the developer experience for implementing the modern title bar.
+
+# Description
+
+The title bar is a fundamental component of a Windows app’s user interface. Here are its key features and functionalities:
+
+1. Display Information:
+    - The title bar prominently displays the name of the app or document currently open. Users can quickly identify which app they are interacting with.
+    - Additionally, the title bar may include other relevant information, such as the document title or the current state (e.g., “Editing,” “Viewing,” etc.).
+2. Window Controls:
+    - Minimize Button: Clicking the minimize button (usually represented by an underscore icon) reduces the app window to the taskbar or system tray.
+    - Maximize Button: Clicking the maximize button (usually represented by a square icon) expands the app window to fill the screen.
+    - Close Button: Clicking the close button (usually represented by an “X” icon) closes the app window.
+3. Drag and Move:
+    - Users can click and drag the title bar to move the app window around the screen. This allows them to position the window wherever they prefer.
+4. Theming and Integration:
+    - The title bar can be customized to match the app’s visual style using Mica theming.
+    - It also integrates with other WinUI controls, such as NavigationView, AutoSuggestBox, and PersonPicture, providing a cohesive user experience.
+
+
+# Is this the right control?
+
+A title bar is a core component of a Windows app, and the shell version of the title bar is shipped with the basic WinUI app template by default. 
+
+Use the WinUI TitleBar when you want further customizations such as subtitles, Mica theming, and integrations with WinUI Controls.
+
+_Note: WinUI TitleBar does not handle Caption Buttons - it simply allocates space on the right side. 
```

Comment:
> on the right side

Here's the dual meaning of "right": correct, or right-hand side? Because in a RTL environment it should be on the left. I assume it handles RTL properly.

Reply:
Good call. Updated.